### DISCUSSION TITLE
Standards for behavioral data?

### DISCUSSION THREAD

**Discussion by KravitzLab** [dandi/helpdesk#51] [General]
Hello!  I develop and maintain a behavioral device called [FED3](https://github.com/KravitzLabDevices/FED3), which writes .csv files of behavioral events during experiments.  I'd like to explore putting FED3 data on Dandi - can you advise me on acceptable or preferred formats for behavioral data?

**Comment by bendichter** [comment]
Sounds great. Could you provide an example output file? Are there certain columns that are always there?

**Reply by KravitzLab** [reply]
Thanks for the quick reply!  I uploaded five example datafiles [here](https://github.com/KravitzLabDevices/FED3/tree/main/ExampleData).  The datafiles from the device always have the same columns as in these files, although I keep adding functionality to the device so additional columns may be added at a later date.

**Reply by bendichter** [reply]
@rly what do you think would be best for this? Maybe your [ndx-events](https://github.com/rly/ndx-events) extension would help?
